src/data/equiv/local_equiv.lean

```diff
@@ -70,6 +70,37 @@ its output. The list of lemmas should be reasonable (contrary to the output of
 enough.
 "
 
+-- register in the simpset `mfld_simps` several lemmas that are often useful when dealing
+-- with manifolds
+attribute [mfld_simps] id.def function.comp.left_id set.mem_set_of_eq set.image_eq_empty
+set.univ_inter set.preimage_univ set.prod_mk_mem_set_prod_eq and_true set.mem_univ
+set.mem_image_of_mem true_and set.mem_inter_eq set.mem_preimage function.comp_app
+set.inter_subset_left set.mem_prod set.range_id and_self set.mem_range_self
+eq_self_iff_true forall_const forall_true_iff set.inter_univ set.preimage_id function.comp.right_id
+not_false_iff and_imp set.prod_inter_prod set.univ_prod_univ true_or or_true
+
+namespace tactic.interactive
+
+/-- A very basic tactic to show that sets showing up in manifolds coincide or are included in
+one another. -/
+meta def mfld_set_tac : tactic unit := do
+  goal ← tactic.target,
+  match goal with
+  | `(%%e₁ = %%e₂) :=
+      `[ext my_y,
+        split;
+        { assume h_my_y,
+          try { simp only [*, -h_my_y] with mfld_simps at h_my_y },
+          simp only [*] with mfld_simps }]
+  | `(%%e₁ ⊆ %%e₂) :=
+      `[assume my_y h_my_y,
+        try { simp only [*, -h_my_y] with mfld_simps at h_my_y },
+        simp only [*] with mfld_simps]
+  | _ := tactic.fail "goal should be an equality or an inclusion"
+  end
+
```

Comment:
For the future, not for this PR -- Is there a way to extend the simp set so that the following example is within reach of `mfld_set_tac`?
```
import topology.local_homeomorph
open set local_homeomorph
variables {H : Type*} [topological_space H]

example (e : local_homeomorph H H) (s : set H) (open_s : is_open s) :
  (e.source ∩ s) ⊆ (e.restr s).source :=
begin
  mfld_set_tac,
  rw interior_eq_of_open open_s, exact h_my_y.2,
end
```

Reply:
Yes, there are definitely improvements that I would like to try on `mfld_set_tac`!